test/instrumentation/modules/@aws-sdk/client-dynamodb.test.js

```diff
@@ -0,0 +1,370 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict';
+
+// Test DynamoDB instrumentation of the '@aws-sdk/client-dynamodb' module.
+//
+// Note that this uses localstack for testing, which mimicks the DynamoDB API but
+// isn't identical. Some known limitations:
+// https://docs.localstack.cloud/user-guide/aws/dynamodb/
+
+if (process.env.GITHUB_ACTIONS === 'true' && process.platform === 'win32') {
+  console.log('# SKIP: GH Actions do not support docker services on Windows');
+  process.exit(0);
+}
+
+const test = require('tape');
+
+const { validateSpan } = require('../../../_validate_schema');
+const { runTestFixtures, sortApmEvents } = require('../../../_utils');
+const { iitmVersionsFrom } = require('../../../testconsts');
+const AWS_REGION = 'us-east-2';
+const LOCALSTACK_HOST = process.env.LOCALSTACK_HOST || 'localhost';
+const endpoint = 'http://' + LOCALSTACK_HOST + ':4566';
+
+const NODE_VER_SUP_AWS_SDK = '14.0.0';
+const NODE_VER_SUP_RANGE_RITM = `>=${NODE_VER_SUP_AWS_SDK}`;
+const NODE_VER_SUP_RANGE_IITM = iitmVersionsFrom(NODE_VER_SUP_AWS_SDK);
```

Comment:
Note for reviewer: this is to avoid testing ESM on Node v12 since the dynamodb client doesn't support it. Check `./test/testconsts.js` module for implementation details.

Reply:
done! also did the replacement in fastify and http tests :)